src/cargo/sources/registry/mod.rs

```diff
@@ -267,6 +267,7 @@ pub struct RegistrySource<'gctx> {
     /// warning twice, with the assumption of (`dep.package_name()` + `--precise`
     /// version) being sufficient to uniquely identify the same query result.
     selected_precise_yanked: HashSet<(InternedString, semver::Version)>,
+    selected_precise_prerelease: HashSet<(InternedString, semver::Version)>,
```

Comment:
A similar comment as yanked one can be added for this field, if we're going to emit that warning